nemo/collections/asr/parts/utils/nmse_clustering.py

```diff
@@ -30,105 +30,91 @@
 # This file is part of https://github.com/scikit-learn/scikit-learn/blob/114616d9f6ce9eba7c1aacd3d4a254f868010e25/sklearn/manifold/_spectral_embedding.py and
 # https://github.com/tango4j/Auto-Tuning-Spectral-Clustering.
 
+import warnings
```

Comment:
replace this with our own nemo logging warnings

Reply:
replaced with nemo logging.warning